sunpy/map/tests/test_mapsequence.py

```diff
@@ -181,3 +184,17 @@ def test_quicklook(mapsequence_all_the_same):
 
         for m in mapsequence_all_the_same.maps:
             assert m._repr_html_() in html_string
+
+
+@figure_test
+@pytest.mark.remote_data
+@pytest.mark.filterwarnings('ignore:.*')
+def test_norm_animator():
+    # To replicate https://github.com/sunpy/sunpy/issues/4763
+    res = Fido.search(a.Instrument.hmi, a.Physobs.los_magnetic_field,
+                      a.Time("2020/01/01", "2020/01/01T00:01:30"))
+    files = Fido.fetch(res)
```

Comment:
can't you just construct a mapsequence out of the hmi test image twice?

Reply:
I will try.